scripts/qesap/test/unit/test_qesap_configure.py

```diff
@@ -203,15 +203,44 @@ def test_configure_without_ansible(base_args, tmpdir):
     config_file_name = str(tmpdir / 'config.yaml')
     with open(config_file_name, 'w', encoding='utf-8') as file:
         file.write(config)
-    os.makedirs(os.path.join(tmpdir, 'terraform'))
     os.makedirs(os.path.join(tmpdir, 'terraform', 'pinocchio'))
 
     args = base_args(base_dir=tmpdir, config_file=config_file_name)
     args.append('configure')
     assert main(args) == 0
 
 
-def test_configure_without_ansible_in_older_apiver(base_args, tmpdir):
+def test_configure_apiver_older_than_3(base_args, tmpdir):
+    config = """---
+apiver: 2
+provider: pinocchio
+terraform:
+  variables:
+    az_region: "westeurope"
+    hana_ips: ["10.0.0.2", "10.0.0.3"]
+    hana_data_disks_configuration:
+      disk_type: "hdd,hdd,hdd"
+      disks_size: "64,64,64"
+ansible: pippo
+"""
+    config_file_name = str(tmpdir / 'config.yaml')
+    with open(config_file_name, 'w', encoding='utf-8') as file:
+        file.write(config)
+    os.makedirs(os.path.join(tmpdir, 'terraform', 'pinocchio'))
+    os.makedirs(os.path.join(tmpdir, 'ansible', 'playbooks', 'vars'))
+
+    args = base_args(base_dir=tmpdir, config_file=config_file_name)
+    args.append('configure')
+    assert main(args) == 1
+
+
+def test_configure_apiver_older_than_3_no_ansible(base_args, tmpdir):
+    '''
+       Test is using th eno more supported apiver:2
```

Comment:
s/Test is using th eno more supported apiver:2/Test is using the unsupported apiver:2/